scene/2d/polygon_2d.cpp

```diff
@@ -222,20 +222,6 @@ void Polygon2D::_notification(int p_what) {
 				}
 
 				for (int i = 0; i < bone_weights.size(); i++) {
-					if (bone_weights[i].weights.size() != points.size()) {
-						continue; //different number of vertices, sorry not using.
```

Comment:
I said remove the prints, not the `continue`. All those need to be **skipped**, but we don't need the spam.

Reply:
Sorry. This is what I get for programming just after I woke up.